pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -20,18 +20,18 @@
 
 import cockpit from 'cockpit';
 import React, { PropTypes } from "react";
-import DialogPattern from 'cockpit-components-dialog.jsx';
-import Select from "cockpit-components-select.jsx";
-import FileAutoComplete from "../../lib/cockpit-components-file-autocomplete.jsx";
-import { createVm, addErrorNotification } from '../actions.es6';
+import DialogPattern from '../../../lib/cockpit-components-dialog.jsx';
+import Select from "../../../lib/cockpit-components-select.jsx";
```

Comment:
Not that important, but I don't see why our pkg/lib/ import path would suddenly stop working just because this file moved one directory level down?